server/resp/src/main/java/org/infinispan/server/resp/Resp3Handler.java

```diff
@@ -70,7 +70,7 @@ public RespRequestHandler handleRequest(ChannelHandlerContext ctx, String type,
                         buf.writeByte('\r').writeByte('\n');
                         ctx.writeAndFlush(buf);
                      } else {
-                        ctx.writeAndFlush(RespRequestHandler.stringToByteBuf("_\r\n", ctx.alloc()));
+                        ctx.writeAndFlush(RespRequestHandler.stringToByteBuf("$-1\r\n", ctx.alloc()));
```

Comment:
We have from [here](https://redis.io/commands/get/) we must return `nil` when a key is not found. And from [here](https://redis.io/docs/reference/protocol-spec/#resp-bulk-strings) that the nil bulk string is `"$-1\r\n"`.